Gemfile.lock

```diff
@@ -60,6 +60,7 @@ GEM
       scrypt (>= 1.2, < 4.0)
     authlogic-oid (1.0.4)
       authlogic
+      hoe (>= 1.12.2)
```

Comment:
Hi @titchimoto the new view is looking good! Could you please remove this `Gemfile.lock` from your PR? Thanks!
It may be of help https://stackoverflow.com/questions/38743912/git-remove-a-file-from-a-pull-request